packages/config/src/projects/layer2s/aevo.ts

```diff
@@ -2,12 +2,14 @@ import { EthereumAddress, UnixTime } from '@l2beat/shared-pure'
 
 import { subtractOne } from '../../common/assessCount'
 import { ProjectDiscovery } from '../../discovery/ProjectDiscovery'
+import { Badge } from '../badges'
 import { CELESTIA_DA_PROVIDER, opStackL2 } from './templates/opStack'
 import { Layer2 } from './types'
 
 const discovery = new ProjectDiscovery('aevo')
 
 export const aevo: Layer2 = opStackL2({
+  badges: [Badge.VM.EVM, Badge.DA.Celestia, Badge.Stack.OPStack],
```

Comment:
pls add conduit as a raas to conduit chains, check either https://rollup.wtf/ or https://app.conduit.xyz/browse-networks